alphagov/paas-product-page
# Replace jQuery 3.2.1 with 3.4.1

What
----

Replace jQuery 3.2.1 with 3.4.1 to keep dependencies up to date to avoid vulnerabilities

How to review
-------------
- check header menu on mobile still expands
- check signup step validation works and back button history still works

Who can review
--------------
not @kr8n3r
